resources/variants/strateo3d_standard_10.inst.cfg

```diff
@@ -0,0 +1,20 @@
+[general]
+name = Standard 1.0 BETA!
```

Comment:
Why add the BETA! in a profile? 

If it's not yet tested, I think it's better to leave it out of an official Cura release. Changing this later on might even cause issues.

Reply:
The profiles for these nozzles have been tested on certain parts but are still being improved to accommodate more complexe prints . We sell 1mm and 1.2mm nozzles and would like to make our profiles available to our clients who are interested in printing with these nozzles. Would changing 'BETA!' to 'Experimental' be acceptable ?
Thanks